test/i965_avce_context_test.cpp

```diff
@@ -228,6 +233,16 @@ TEST_P(AVCEContextTest, QualityRange)
         {VAProfileH264MultiviewHigh, 1u},
         {VAProfileH264StereoHigh, 1u},
     };
+    const std::map<VAProfile, unsigned> qranges_skl_bxt = {
+        {VAProfileH264ConstrainedBaseline, entrypoint == VAEntrypointEncSliceLP
+            ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
+        {VAProfileH264Main, entrypoint == VAEntrypointEncSliceLP
+            ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
+        {VAProfileH264High, entrypoint == VAEntrypointEncSliceLP
+            ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
+        {VAProfileH264MultiviewHigh, ENCODER_QUALITY_RANGE_AVC},
+        {VAProfileH264StereoHigh, ENCODER_QUALITY_RANGE_AVC},
+    };
 
```

Comment:
Perhaps just declare one map instead of two and assign its values conditionally...

```
std::map<VAProfile, unsigned> qranges;
if (is_skl_bxt) {
       qranges = {
            {VAProfileH264ConstrainedBaseline, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
            {VAProfileH264Main, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
            {VAProfileH264High, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE_AVC},
            {VAProfileH264MultiviewHigh, ENCODER_QUALITY_RANGE_AVC},
            {VAProfileH264StereoHigh, ENCODER_QUALITY_RANGE_AVC},
        };
} else {
      qranges = {
            {VAProfileH264ConstrainedBaseline, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE},
            {VAProfileH264Main, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE},
            {VAProfileH264High, entrypoint == VAEntrypointEncSliceLP
                ? ENCODER_LP_QUALITY_RANGE : ENCODER_QUALITY_RANGE},
            {VAProfileH264MultiviewHigh, 1u},
            {VAProfileH264StereoHigh, 1u},
        };
}